src/main/kotlin/lotto/domain/LottoMachine.kt

```diff
@@ -1,16 +1,19 @@
 package lotto.domain
 
-class LottoMachine(private val lottoPrice: Int, private val lottoGenerator: LottoGenerator) {
+class LottoMachine(private val lottoPrice: Money, private val lottoGenerator: LottoGenerator) {
 
-    init {
-        require(lottoPrice > 0) { "로또금액은 0보다 커야합니다" }
-    }
+    fun sellLottos(money: Money, manualLottoGenerators: List<ManualLottoGenerator> = emptyList()): List<Lotto> {
```

Comment:
인터페이스를 타입으로 사용하는 습관을 길러 두면 프로그램이 훨씬 유연해질 거예요. 인터페이스와 다형성에 대해 알아보고 아래의 글도 참고하면 좋을 것 같아요.
https://stackoverflow.com/questions/11683044/return-arraylist-or-list/11683073

```suggestion
    fun sellLottos(money: Money, lottoGenerators: List<LottoGenerator> = emptyList()): List<Lotto> {
```

Reply:
인터페이스를 타입으로 사용하도록 변경해보았어요.

그리고, 파라미터명에는 수동/랜덤 이라는 네이밍을 남겨두었는데요. 8e66ea5

수동/랜덤 이라는 네이밍이 없다면 lottoGenerator에 대해 의미를 파악하기 어려울 것이라 생각했기 때문이에요.
또한, 수동/랜덤이라는 개념은 쉽게 변경되지 않을 것아 변수명으로 남겨보았어요.